src/main/java/com/textmessenger/controller/UserController.java

```diff
@@ -41,6 +41,12 @@ public ResponseEntity endpoint() {
     }
   }
 
+  @DeleteMapping("/current")
+  public void deleteCurrent() {
+    userEndPoint = null;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make the enclosing method "static" or remove this set. ([squid:S2696](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS2696))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=138&issues=AWVoAa90XKENQn1Sh-NU&open=AWVoAa90XKENQn1Sh-NU)

[ISSUE_KEY]: AWVoAa90XKENQn1Sh-NU